src/services/sendEmail.service.js

```diff
@@ -0,0 +1,47 @@
+
+/* eslint-disable require-jsdoc */
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+import mailGen from 'mailgen';
+
+dotenv.config();
+sgMail.setApiKey(process.env.MY_DEV_KEY);
+
+export default class EmailSender {
+  static sendSignUpVerificationLink = async (toEmail, link) => {
+    const mailGenerator = new mailGen({
+      theme: 'default',
+      product: {
+        name: 'Barefoot Nomad',
+        link: '#'
+      }
+    });
+
+    const generateEmail = async () => ({
+      body: {
+        name: 'Successful signup verification email',
+        intro: 'Congractulations, you have sucessfully created the account!',
```

Comment:
I think you wanted to write `Congratulation`

Reply:
@muchristian You got it wrong. that is the template that will be sent to the Email